docs/Queryable-Extensions.md

```diff
@@ -208,7 +208,15 @@ You may also use a dictionary to build the projection values:
 dbContext.Courses.ProjectTo<CourseModel>(Config, new Dictionary<string, object> { {"currentUserName", Request.User.Name} });
 ```
 
-However, using a dictionary will result in hard-coded values in the query instead of a parameterized query, so use with caution.
+However, using a dictionary will result in hard-coded values in the query instead of a parameterized query, so use with caution.
+
+### Recursive queries
+
+Ideally, you would avoid queries that call themselves (do some research). But if you must, you need to enable them:
+
+```c#
+configuration.Internal().RecursiveQueriesMaxDepth = someRandomNumber;

```

Comment:
The default is zero. We could choose a number :)

Reply:
One? Three? Five? Whatever we pick people will complain.

Reply:
Word :)